consensus/core/src/block_manager.rs

```diff
@@ -310,10 +305,7 @@ impl BlockManager {
         &mut self,
         unsuspended_blocks: impl IntoIterator<Item = VerifiedBlock>,
     ) -> Vec<VerifiedBlock> {
-        let (gc_enabled, gc_round) = {
-            let dag_state = self.dag_state.read();
-            (dag_state.gc_enabled(), dag_state.gc_round())
-        };
+        let gc_round = self.dag_state.read().gc_round();
```

Comment:
`consensus_median_based_commit_timestamp` has rolled out now. Is the plan to remove `verify_block_timestamps()` in another PR?

Reply:
I am planning to do on an upcoming PR